### PR TITLE
KFSTP-1998

### PR DESCRIPTION
**Pull request by smith750**
Explicitly list all staging directories so that startup will automatically create them

### CODE REVIEW

File: src/main/resources/org/kuali/kfs/module/tem/spring-tem.xml [hunk 1]
```diff
@@ -68,7 +68,12 @@
 		<property name="batchFileDirectories">
 			<list>
 				<value>${reports.directory}/tem</value>
-				<value>${staging.directory}/tem</value>
+				<value>${staging.directory}/tem/agencyData</value>
+				<value>${staging.directory}/tem/creditCardData</value>
+				<value>${staging.directory}/tem/error</value>
+				<value>${staging.directory}/tem/federalPerDiemTxt</value>
+				<value>${staging.directory}/tem/federalPerDiemXml</value>
+				<value>${staging.directory}/tem/profileExport</value>
```
Comment (bhutchinson) [inline src/main/resources/org/kuali/kfs/module/tem/spring-tem.xml]: is the whitespace really funky here, or is github just confused?

Comment (smith750) [inline src/main/resources/org/kuali/kfs/module/tem/spring-tem.xml]: I'm having trouble fixing, so...I'm thinking the whitespace really is funky for some reason.  Thoughts?

Comment (kellerj) [inline src/main/resources/org/kuali/kfs/module/tem/spring-tem.xml]: It looks like it's ok.  You just need to stretch the browser window wide enough so it doesn't wrap.

Comment (bhutchinson) [inline src/main/resources/org/kuali/kfs/module/tem/spring-tem.xml]: Ok, I'd say let it go then, probably not worth too much time/stress. Thanks.

Comment (smith750) [inline src/main/resources/org/kuali/kfs/module/tem/spring-tem.xml]: Cool.  Thank you both!

**Pull request merged**